server/app/api/v1/endpoints/rooftops.py

```
@@ -165,6 +166,20 @@ async def update_rooftop(
+                db=db,
+                organization_id=organization.id,
+                rooftop_id=rooftop_id,
+            )
+            if not in_sub:
+                raise HTTPException(
+                    status_code=status.HTTP_403_FORBIDDEN,
+                    detail=(
+                        "Overage limit can only be changed when this rooftop is part of an active subscription. "
+                        "Add the rooftop to a subscription first."
+                    ),
+                )
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)